test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -5259,6 +5262,14 @@ public void PackCommand_PackProjectWithCentralTransitiveDependencies()
                         "ManagePackageVersionsCentrally",
                         "true");
 
+                    if (transitiveDependencyPinningEnabled != null)
+                    {
+                        ProjectFileUtils.AddProperty(
+                            xml,
+                            "TransitiveDependencyPinningEnabled",
+                            transitiveDependencyPinningEnabled);
+                    }
+
```

Comment:
Let's update this to also use `ProjectBuildProperties`
```suggestion
                        ProjectBuildProperties.ManagePackageVersionsCentrally,
                        "true");

                    if (transitiveDependencyPinningEnabled != null)
                    {
                        ProjectFileUtils.AddProperty(
                            xml,
                            ProjectBuildProperties.TransitiveDependencyPinningEnabled,
                    }

```